src/components/WhoAreWe.astro

```diff
@@ -0,0 +1,29 @@
+<div class="mx-auto max-w-lg">
+  <h1 class="text-6xl">Who Are We</h1>
+
+  <div class="flex flex-wrap p-4 gap-4">
+    <p class="text-sm">
```

Comment:
Should be text-xl according to the design (20 pt)

https://tailwindcss.com/docs/font-size

Reply:
fixed

Reply:
fixed